test/AutomationNotes.MD

```diff
@@ -0,0 +1,38 @@
+# AUTOMATION NOTES:
```

Comment:
What about `test/README.md`? Then it's easily visible on github when you go to the `test` folder